src/decorator-annotator.ts

```diff
@@ -189,13 +191,19 @@ export class DecoratorClassVisitor {
     }
   }
 
-  isClassDecorator(decorator: ts.Decorator): boolean {
+  /**
+   * Checks if the decorator is on a class, as opposed to a field or an
```

Comment:
your editor seems to be wrapping at 80, is that intentional? Shouldn't it be 100?

Reply:
I insert the line breaks manually.